montreal-python/videos/nhl-goal-celebration-hack-with-light-show-and-machine-learning-mp55.json

```diff
@@ -1,41 +0,0 @@
-{
-  "copyright_text": "Creative Commons Attribution license (reuse allowed)",
-  "description": "Montreal, Nov. 23, 2015 - In this talk Fran\u00e7ois Maillet explains how he trained a machine learning model in detecting in real-time that a goal was just scored by the Montreal Canadiens hockey team based on the live audio feed of a game, and triggering a light show using Philips hues in his living room. The system was built using various Python modules, more specifically scikit-learn, pyaudio, librosa, phue and bottle.\n\nRecorded and edited live by @ChristianAubry at Montr\u00e9al Python 55 (MP55: Wagnerian Xenosaurus) with a little big help from Savoir-faire Linux (http://sflx.ca/jobs).\n\n* About Fran\u00e7ois : https://twitter.com/mailletf\n* About the hack : http://sflx.ca/EpicCelebration\n* About MP55: http://sflx.ca/MP55e",
-  "duration": 1534,
-  "language": "eng",
-  "recorded": "2015-11-23",
-  "related_urls": [
-    {
-      "label": "group web",
-      "url": "https://montrealpython.org"
-    },
-    {
-      "label": "MP55",
-      "url": "http://sflx.ca/mp55"
-    },
-    {
-      "label": "Epic Celebration blog entry",
-      "url": "http://sflx.ca/EpicCelebration"
-    },
-    {
-      "label": "Fran\u00e7ois Maillet twitter",
-      "url": "https://twitter.com/mailletf"
-    },
-    {
-      "label": "Savoir-faire Linux",
-      "url": "http://sflx.ca/jobs"
-    }
-  ],
-  "speakers": [
-    "Fran\u00e7ois Maillet"
-  ],
-  "tags": [],
-  "thumbnail_url": "https://i.ytimg.com/vi/KU86lwojGl0/maxresdefault.jpg",
-  "title": "NHL goal celebration hack with light show and machine learning",
```

Comment:
This (and more) talk have title.
Is it an unintended deletion?

Reply:
Yeah, I'm gonna update the description of the PR with more details mostly that is because github didn't understand that i've renamed this file to:

`mp55-nhl-goal-celebration-hack-with-light-show-and-machine-learning.json`

Although we don't have the notion of season, every meetup has a number which we are calling `mp`, i'm then cleaning the list of talks to include the meeting number into the title, that's gonna help the list of talks to be ordered properly.

Reply:
This is OK for an `ls`. If you want to have it sorted in pyvideo they need to have a correct `recorded` field. It's displayed in descending chronological order.

Reply:
yup that's a bit i though.

I'm also gonna make sure the metadata is gonna have those informations. Is it being by pyvideo by the way (the metadata)?

Reply:
> Is it being by pyvideo by the way (the tags)?

@mlhamel , I don't understand the question.

Reply:
Lost in translation too. Is it about `tags` or `recorded`?
* `tags` are pending in Montréal-Python
* `recorded` should be OK, displayed in descending chronological order in http://pyvideo.org/events/montreal-python.html